src/akkudoktoreos/utils/visualize.py

```diff
@@ -0,0 +1,497 @@
+import os
+from collections.abc import Sequence
+from typing import Callable, Optional, Union
+
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.backends.backend_pdf import PdfPages
+
+from akkudoktoreos.config.config import get_config
+
+
+class VisualizationReport:
```

Comment:
inherit ConfigMixin and then just use self.config (and drop get_config)

Reply:
done